website/server/models/user/schema.js

```diff
@@ -9,6 +9,8 @@ import {
 
 const Schema = mongoose.Schema;
 
+const INVALID_DOMAINS = ['habitica.com', 'habitrpg.com'];
```

Comment:
Do you think there will be more invalid domains eventually? If the list here grows, perhaps exporting stuff to a config file of sorts would be useful. Perhaps also over-engineering 😉